components/zcash_address/src/kind/unified.rs

```diff
@@ -34,7 +35,7 @@ pub enum Typecode {
     P2sh,
     Sapling,
     Orchard,
-    Unknown(u8),
+    Unknown(u64),
```

Comment:
We should double-check that ZIP 316 is clear on whether it allows the full CompactSize space, or whether it is retaining the Zcash consensus rule variant with a limit below a `u32` (in which case this could be a `u32`). Open an issue for this.

Reply:
https://github.com/zcash/zips/pull/551#pullrequestreview-745416777